build_tools/benchmarks/common/benchmark_suite_test.py

```diff
@@ -184,7 +184,8 @@ def test_load_from_run_configs(self):
         module_execution_config=exec_config_a,
         target_device_spec=device_spec_a,
         input_data=common_definitions.ZEROS_MODEL_INPUT_DATA,
-        tool=iree_definitions.E2EModelRunTool.IREE_BENCHMARK_MODULE)
+        tool=iree_definitions.E2EModelRunTool.IREE_BENCHMARK_MODULE,
+        tags=[])
```

Comment:
No need to specify the empty tags if we give the parameter a default value, see the comment below